pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -317,7 +348,7 @@ func (l *LibvirtDomainManager) asyncMigrate(vmi *v1.VirtualMachineInstance) {
 			defer migrationProxy.StopListening()
 			err := migrationProxy.StartListening()
 			if err != nil {
-				l.setMigrationResult(vmi, true, fmt.Sprintf("%v", err))
+				l.setMigrationResult(vmi, true, fmt.Sprintf("%v", err), "")
```

Comment:
we still have setMigrationResult being called from multiple goroutines. I thought the monitor go-routine was going to be the source of truth for the final migration result.

Reply:
This is happening before the monitor has been started.

Reply:
the one at the bottom of this function is after the monitor go routine is launched.

Reply:
ok, removed. I wanted to keep a definitive result here, but it works without it.